internal/cmd/gocdk/biome.go

```diff
@@ -246,17 +263,91 @@ func biomeApply(ctx context.Context, pctx *processContext, biome string, input b
 		return xerrors.Errorf("biome apply %s: %w", biome, err)
 	}
 
-	c := pctx.NewCommand(ctx, biomePath, "terraform", "init", "-input="+strconv.FormatBool(input))
-	if err := c.Run(); err != nil {
+	writeTerraformOutput := func(cmd *exec.Cmd, msg []byte) {
+		const separator = "#########################################\n"
+		pctx.stderr.Write([]byte("\n"))
```

Comment:
Feedback on including this? I thought it would be useful to include the exact command line so the user can reproduce it manually, but maybe only with `--verbose`?

Reply:
I think the command line is helpful.

On a different note, I think I would rather that the Terraform command is streamed to stdout in the case where `--verbose` is passed, so that the user can see the progress.

Reply:
Done. I created a local helper function to exec the command and return the buffered output (if any; in Verbose mode it is `nil`) and the error.